### PR TITLE
Kvng james ng patch 1

### PR DESCRIPTION
**Pull request by KvngJamesNG**
CODEOWNERS V4
As instructed

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,54 @@
+                          ################################################################################
+                          #          Each line is a file pattern followed by our owner/teams             #
+                          ################################################################################
+
+# This is a comment and will be ignored.
+
+# Global owner for everything in this repo
+* @rt4914
+
+# Owners/teams for backend (Ruby on Rails)
```
Comment (rt4914) [inline CODEOWNERS:10]: Remove everyone below this, we just need the global owner, when required I will add things.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,54 @@
+                          ################################################################################
+                          #          Each line is a file pattern followed by our owner/teams             #
+                          ################################################################################
+
+# This is a comment and will be ignored.
```
Comment (rt4914) [inline CODEOWNERS:5]: Remove this.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@KvngJamesNG Thanks for the PR.

1. Have a look at https://github.com/rt4914/KlinicCon-Backend/wiki#creating-a-pull-request espeically point 6 and 7. Please make sure you request a review and assign the PR. to the person who needs to take action.
2. Please update the title to something meaningful: `Add Codeowners for the project`
3. Similarly update PR description too and also mention `Closes #<issue-number>` to mention the issue which is linked to the PR.
4. For the first PR its fine to create a patch, but in future always make sure to create a proper branch and pull-request on this repo similar to the above link.

File: CODEOWNERS (hunk 1 above)
Comment (KvngJamesNG) [reply CODEOWNERS:10]: I am on it

**Pull request closed without merging**